chapter_08_prettification.asciidoc

```diff
@@ -822,6 +859,7 @@ All that took me a few goes, but I'm reasonably happy with it now
 [[homepage-looking-better]]
 .The lists page, looking good enough for now...
 image::images/prettified-final.png["Screenshot of lists page in light mode with decent styling."]
+// CSANAD:  This image seemed to be broken, so I uploaded a new one.
```

Comment:
Please check the resolution, my image ended up significantly smaller in size.

Reply:
thanks Csanad!